specification/paths/Shipments.json

```diff
@@ -217,34 +179,16 @@
             "additionalProperties": false,
             "properties": {
               "data": {
-                "oneOf": [
+                "allOf": [
```

Comment:
You can remove this allOf completely, since the part with `id` readOnly true is already in the Shipment schema.